src/db/collections/outputs/mod.rs

```diff
@@ -365,3 +368,419 @@ impl MongoDb {
         }
     }
 }
+
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct OutputAnalyticsResult {
+    pub count: u64,
+    pub total_value: String,
+}
```

Comment:
Analogous to the `BlocksAnalytics`, I would love to see a combined response here:

```rust
pub struct Amount(String)

pub struct OutputStatistics {
    pub count: Amount,
    pub total_amount: Amount,
}

struct OutputsAnalyticsReponseForLackOfBetterName {
    pub combined: OutputAnalytics,
    pub alias_outputs: OutputAnalytics,
    pub basic_outputs: OutputAnalytics,
    // ...
}
```

Reply:
This would mean creating a facet query (which would not be able to use indexes) or simply calling all of them back-to-back.